rubocop_abcsize.yml

```diff
@@ -0,0 +1,1318 @@
+Inspecting 971 files
```

Comment:
Do we want to commit this one?

Reply:
Yeah, I'm not sure exactly what this is? It looks like rubocop output, what does it being in a file mean/do?